stacks-signer/src/tests/conf/signer-0.toml

```diff
@@ -4,3 +4,4 @@ endpoint = "localhost:30000"
 network = "testnet"
 auth_password = "12345"
 db_path = ":memory:"
+metrics_endpoint = "0.0.0.0:9090"
```

Comment:
The node uses `prometheus_bind`, I suggest keeping the setting name the same for both

Reply:
Phooey. I much prefer "metrics_endpoint" because I have no idea what "prometheus_bind" means but also understand the keeping it consistent XD

Reply:
This PR also implements an "info" endpoint, which responds with generalized JSON for (non-metric) signer monitoring purposes. So, in that case it's maybe better to have `monitoring_bind`?

I'm fine with consistency too, I don't have a very strong feeling about this. If you know what prometheus, `prometheus_bind` makes total sense.